src/database/models/article.js

```diff
@@ -1,33 +1,46 @@
 module.exports = (sequelize, DataTypes) => {
-  const Article = sequelize.define(
-    'Article',
-    {
-      title: DataTypes.STRING,
-      body: DataTypes.TEXT,
-      image: DataTypes.STRING,
-      published: DataTypes.BOOLEAN,
-      publisheddate: {
-        type: DataTypes.DATE,
-        field: 'published_date',
-        defaultValue: new Date()
-      },
-      status: {
-        type: DataTypes.ENUM('draft', 'active', 'deactivated'),
-        defaultValue: 'draft'
-      },
-      authorId: DataTypes.INTEGER,
-      slug: DataTypes.STRING,
-      description: DataTypes.STRING
+  const Article = sequelize.define('Article', {
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: [2]
+      }
     },
-    {}
-  );
+    description: DataTypes.STRING,
```

Comment:
Could you maintain this article model as it was previously so we can only introduce the one that differ from yours as agreed? Also I noticed you changed the foreign key to `id` could we keep it as `authorId` since that is referring to the publisher of the article, this table already has its own `id` column and a lot of methods probably depend on it

Reply:
ok